frontend/device/android/powerd.lua

```diff
@@ -2,18 +2,48 @@ local BasePowerD = require("device/generic/powerd")
 local _, android = pcall(require, "android")
 
 local AndroidPowerD = BasePowerD:new{
-    fl_min = 0, fl_max = 25,
-    fl_intensity = 10,
+    fl_min = 0, fl_max = 100, -- consistant scale across android, kobo ...
+--    fl_intensity = 10,
```

Comment:
is fl_intensity unused?, what's the reason to comment it out?. If is unused and not needed please remove it first.

Reply:
As I see, the fl_intensity is only used on Kobos and Kindles. 
* In Kobos powerd it is initialized in the init().
* On Kindle its a bit more complicated.
* It is also in generic, but there it is also nil.
* In `devicelistener.lua` it is checked, if fl_intensity~=nil, so no problem there.

I will remove it.